packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1502,7 +1502,7 @@ export default abstract class SqlIntegration
           anonymousIdColumn: "user_pseudo_id",
           getMetricTableName: () => this.generateTablePath("events_*"),
           getDateLimitClause: (start: Date, end: Date) =>
-            `_TABLE_SUFFIX BETWEEN '${formatDate(
+            `REGEXP_EXTRACT(_TABLE_SUFFIX, r'[0-9]+') BETWEEN '${formatDate(
```

Comment:
To keep the logic simple and easy to read, switch to using OR and no function calls:

```sql
(
  _TABLE_SUFFIX BETWEEN '...' AND '...' 
  OR _TABLE_SUFFIX BETWEEN  'intraday_...' AND 'intraday_...'
)
```